src/components/WalletModal/index.tsx

```diff
@@ -163,16 +162,12 @@ const sendAnalyticsEventAndUserInfo = (
     wallet_type: walletType,
     is_reconnect: isReconnect,
   })
-  const currentDate = new Date().toISOString()
```

Comment:
Will the date now be captured by Amplitude as metadata for the property (eg when it was set), or do we lose the date by no longer setting it?

Reply:
no we lose the date, its not in the metadata from what I understand. we just see the latest values for users. we can see when user model values changed for each event, and use the dates from the events though.